apps/docs/.eslintrc.json

```diff
@@ -9,23 +9,26 @@
         "@next/next/no-html-link-for-pages": "off"
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/c6a508d8-4926-4334-a8bf-e36786d475c6)

These are barely visible at it's gray on gray.

Reply:
that's what it's all about!